packages/venia-concept/src/components/Header/header.js

```diff
@@ -44,21 +49,18 @@ class Header extends Component {
                         </NavTrigger>
                     </div>
                     <div className={classes.secondaryActions}>
-                        <button className={classes.searchTrigger}>
+                        <SearchTrigger
+                            searchOpen={searchOpen}
+                            toggleSearch={toggleSearch}
+                        >
                             <Icon name="search" />
-                        </button>
+                        </SearchTrigger>
                         <CartTrigger>
                             <Icon name="shopping-cart" />
                         </CartTrigger>
                     </div>
                 </div>
-                <div className={classes.searchBlock}>
-                    <input
-                        className={classes.searchInput}
-                        type="text"
-                        placeholder="I'm looking for..."
-                    />
-                </div>
+                <SearchBar isOpen={searchOpen} classes={classes} />
```

Comment:
This doesn't look right. Are you passing the `classes` object from `header.css` straight through to `SearchBar`?

Reply:
Good catch, totally not needed.